crm_properties.go

```diff
@@ -0,0 +1,106 @@
+package hubspot
+
+import (
+	"fmt"
+)
+
+const (
+	crmPropertiesPath = "properties"
+)
+
+type CrmPropertiesList struct {
+	Results []*CrmProperty
+}
+
+type CrmProperty struct {
```

Comment:
Ahh, I based my code on the sample response assuming it would include all fields..

I've audited this entire object, added the missing fields,  and the fields in the struct should be in the same order as the docs to make this easier to compare...